doc/source/programs/gdal_rasterize.rst

```diff
@@ -191,11 +191,12 @@ raster data is only supported since GDAL 2.1.0.
 
 .. option:: -optim {AUTO|VECTOR|RASTER}
 
-    Force the algorithm used (results are identical). The raster mode is used in most cases and
-    optimise read/write operations. The vector mode is useful with a decent amount of input
-    features and optimise the CPU use. That mode have to be used with tiled images to be
-    efficient. The auto mode (the default) will chose the algorithm based on input and output
-    properties.
+    Force the algorithm used (results are identical). Raster mode
+    is used in most cases and  optimizes read/write  operations.  The
+    vector mode is useful with a large amount of input features and
+    optimizes CPU use, provided that the input image is tiled.
+    Auto mode (the default) will chose the
```

Comment:
output image

Reply:
didn't understand your first suggestion ("-> that mode has to be used"). Fixes for other issues in https://github.com/OSGeo/gdal/pull/11775

Reply:
It refers to the error in English in "That mode **have** to be used with tiled images..." but it is now gone anyway.